ROLES.md

```diff
@@ -156,6 +156,16 @@ the area before being accepted into the source base.
 
 Members who frequently contribute code are expected to proactively perform code reviews for the area that they are active in.
 
+### Ongoing membership
+
+Members are expected to be active participants in the project. If an individual doesn't contribute to the project for 
+a 180 day period, than that individual may lose membership. Contributing to the project can be done in a number of ways:
+
+- Merging pull requests
```

Comment:
Is sending a PR good enough or is it only the merging? And presumably we mean being the author of a merged PR, not the actual merge (which is done by a bot anyway?).

Reply:
yes, only admin has permission to merge afaik, would be good to clarify this is submit PRs that are merged.